### ISSUE TITLE
Wrong links in task output

### ISSUE THREAD

**Issue by DenisBalan** [trstringer/manual-approval#136]
We are using this task 'trstringer/manual-approval' for manual approval process in enterprise server.

in output can see different urls, mixed formats, either pointing to github.com or to local endpoint, but not clickable because it drives you to api link..

This issue is to address those links formats and make them clickable.

Thx.

![image](https://github.com/user-attachments/assets/c4763dbf-0991-4539-b124-c8d7d80d172c)

**Comment by DenisBalan** [comment]
In issue description also wrong links, should point to local Github Enterprise Server

![{17D0E03E-11AB-46B1-A67B-4520F0CF21A6}](https://github.com/user-attachments/assets/4aa8ab2a-3163-4479-abe3-4fc810a72dce)

**Comment by yuqu** [comment]
The issue should have been fixed with https://github.com/trstringer/manual-approval/pull/125 and https://github.com/trstringer/manual-approval/pull/112. 

@trstringer we need a release to see the changes on `trstringer/manual-approval@v1`

@DenisBalan alternatively you may point your action to main branch (`trstringer/manual-approval@main`) until a release is published.

**Comment by martian-str33t** [comment]
@trstringer Could we release a new version with the fix?

**Comment by matt-horwood-mayden** [comment]
@trstringer please progress this

also linked to #128

**Comment by venkatamutyala** [comment]
@trstringer 🙏🏾

**Comment by adHosta** [comment]
Yeah I'm also seeing the same issue with the wrong link being provided. 
The "api" prefix should be removed from the URL link.

**Comment by lzhou-aops** [comment]
Would love to see this fix released as well!

**Comment by ClaytonHughes** [comment]
@yuqu I don't believe those commits fix the issue. If I pin the sha containing those commits, I still get see the api.github.com url in the issue body.

For reference, the (sanitized) workflow step configuration:

```yaml
# jobs.*.steps:
    - name: Manual Gate
      if: ${{ env.deploy_environment == 'QA' }}
      timeout-minutes: 15
      # pin: two url bugfixes not yet included in a release (as of 1.9.1)
      uses: trstringer/manual-approval@d2e412d84000579996fd00d7834a817f92af19d3 # or 51b76c4576aad25fefe130f04f4cc4de75814e94, same result
      with:
        secret: ${{ steps.team-approval-auth.outputs.token }}
        approvers: qa-approvers
        minimum-approvals: 1
        issue-title: "Deploying to ${{ env.deploy_environment }}"
        issue-body: >
          Please approve or deny the deployment to
          ${{ env.github_environment }}
          from ${{ github.ref_type }} ${{ github.ref_name }}
        exclude-workflow-initiator-as-approver: true
        additional-approved-words: ''
        additional-denied-words: ''
```

And the created Issue:

Title:
>Manual approval required for workflow run 12345678901: Deploying to QA

Body:
>Please approve or deny the deployment to QA from branch fix/manual-approval-link
>
>
>Workflow is pending manual review.
>URL: https://api.github.com/ORG/REPO/actions/runs/12345678901
>
>Required approvers: [usera userb userc]
>
>Respond "approved", "approve", "lgtm", "yes" to continue workflow or "denied", "deny", "no" to cancel."

**Comment by dan-buzan-mesh** [comment]
Hello, do you have an update when the new release will be out?

**Comment by ddarbyson** [comment]
Waiting on the release to fix the issues identified in their thread too.

**Comment by lizziemac** [comment]
duplicate of https://github.com/trstringer/manual-approval/issues/126, but since this has more user feedback, will close the other one

**Comment by swirle13** [comment]
@lizziemac are you working on the 2.0 release? Would love to help contribute to get this moved along.

Bad issue url: `https://api.github.com/repos/ORG/REPO/issues/123`  
Good issue url: `https://github.com/ORG/REPO/issues/123`

**Comment by venkatamutyala** [comment]
> [@lizziemac](https://github.com/lizziemac) are you working on the 2.0 release? Would love to help contribute to get this moved along.
> 
> Bad issue url: `https://api.github.com/repos/ORG/REPO/issues/123` Good issue url: `https://github.com/ORG/REPO/issues/123`

@SnskArora @caruccio @beverts312 @fnkr I see all of you have managed to get commits into `main`. would you be able to assist with this issue?

**Comment by aantia** [comment]
I didn't see any specific mention of this in the linked threads, so I'd like to add that the issue also affects fetching approver teams e.g. I see in my output `Attempting to expand user orgname/teamname as a group (may not succeed)
GET https://api.github.com/orgs/ccs-group/teams/teamname/members: 404 Not Found []`

**Comment by jpaovivo** [comment]
this one works well:
https://github.com/marketplace/actions/approve-workflow-manual

**Comment by benpriestman** [comment]
A fresh release would be lovely. There's lots of good feedback from ITV about this action. It's just the approval links, that are causing us problems. Anything we can do to help?

**Comment by rdi-robert-uc** [comment]
Hey @lizziemac any estimates about the new release? Thanks in advance.

**Comment by snskArora** [comment]
This is resolved now, closing the issue. If such an issue pops back, please consider opening a new issue
